slug.go

```diff
@@ -539,8 +548,15 @@ func (p *Packer) validSymlink(root, path, target string) (bool, error) {
 		if !strings.HasSuffix(prefix, "/") {
```

Comment:
String manipulation here after a clean may cause some issues that we faced before that was resolved by https://github.com/hashicorp/go-slug/commit/d8b1ee7171000f15726d9e0583d88132232b9229. The `filepath.Join` function should already handle missing `"/"`. Some tests would be helpful on here for this.

Reply:
https://github.com/hashicorp/go-slug/pull/80/commits/63afbb8c6015d9f3ebdee18902886742ef0170f6 should cover it. Since the change introduces `filepath.Join` we don't need the additional prefix.